scripts/build.ps1

```diff
@@ -800,6 +801,12 @@ function Create-VsixPackage
     # Copy CUIT Related depedencies
     $legacyDir = Join-Path $env:TP_PACKAGES_DIR "Microsoft.VisualStudio.CUIT\$testPlatformExternalsVersion\tools\net451"
     Copy-Item -Recurse $legacyDir\* $packageDir -Force
+    $fileToRemove = Join-Path $packageDir "Microsoft.VisualStudio.Ole.Interop.dll"
+    Remove-Item $fileToRemove -Force
```

Comment:
This should be temporarily code for removal. When we release new CUIT nuget package, which does not contains this old interop, so it will not appear in artifacts, so no need to remove it.

Reply:
Agreed, it's temporary until we fix CUIT. But we can fix it later though as the priority is to have the updated interops for the insertion, so we should keep this here for now.